user-service/src/main/java/rs/edu/raf/banka1/bootstrap/BootstrapData.java

```diff
@@ -96,11 +98,16 @@ public BootstrapData(
         this.employeeService = employeeService;
         this.orderRepository = orderRepository;
         this.transferService = transferService;
+        this.contractRepository = contractRepository;
     }
 
     @Override
     public void run(String... args) {
- /*
+
+        if(employeeRepository.findByEmail("admin").isPresent()) {
+            return;
+        }
```

Comment:
Je l ovo provera zato što nam je strategija `UPDATE`, pa nema potrebe da stalno punimo bazu?

Reply:
Izvini, sad sam video da si na discord-u veće rekao razlog.